hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -785,14 +785,15 @@ public void setOwner(final Path path, final String owner, final String group) th
               "This operation is only valid for storage accounts with the hierarchical namespace enabled.");
     }
 
-    LOG.debug(
-            "setOwner filesystem: {} path: {} owner: {} group: {}",
-            client.getFileSystem(),
-            path.toString(),
-            owner,
-            group);
-
     try (AbfsPerfInfo perfInfo = startTracking("setOwner", "setOwner")) {
+
+      LOG.debug(
```

Comment:
Why are we moving it inside of the perfInfo?
It doesn't look like we are even using anything from there.

Reply:
This applies to all the others, I might be missing something.

Reply:
BTW, we don't need the toString() in the path.

Reply:
@goiri For the sake of consistency and of being objective, we should consider each member method here as a black box. Wrapping the contents of each method inside the instrumentation tracker is the overall objective here. This essentially helps us get a picture that is closer to what the users of the ABFS driver will see.
On the request of @DadanielZ, the lines for the check of getIsNamespaceEnabled() is outside of this wrapper but we should treat that as an exception.

Reply:
@goiri for the toString() in the path: I can remove the toString() in a subsequent PR. Are you okay with that?

Reply:
That makes sense.
For the toString(), yes please open a follow up JIRA for that.

Reply:
@goiri Follow-up JIRA is here: https://issues.apache.org/jira/browse/HADOOP-16682